crates/biome_css_analyze/tests/specs/nursery/noIrregularWhitespace/invalid.css.snap

```diff
@@ -394,14 +394,14 @@ invalid.css:85:11 lint/nursery/noIrregularWhitespace ━━━━━━━━━
 ```
 
 ```
-invalid.css:95:11 lint/nursery/noIrregularWhitespace ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+invalid.css:95:2 lint/nursery/noIrregularWhitespace ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
   ! Irregular whitespace found.
   
     93 │ }
     94 │ /* \u{3000} */
   > 95 │ @container␠(width < 15rem) {
-       │           ^
+       │  ^^^^^^^^^^
```

Comment:
This is a regression 😔

Reply:
Indeed. This is strange that this affects this rule. I will have to investigate why.